notebook/notebookapp.py

```diff
@@ -240,15 +240,15 @@ def init_handlers(self, settings):
         
         # BEGIN HARDCODED WIDGETS HACK
         try:
-            import ipywidgets
+            widgets = __import__('jupyter-js-widgets-nbextension')
```

Comment:
Well, that will make the notebook not work with current ipywidgets. Won't it ?

and `jupyter-js-widgets-nbextension` feel weird. It's likely **not* a valid python package name,. You shouldn't **have to** use import.